ruby/ruby-agreement.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width">
+<title>Agreement on HTML Ruby Markup</title>
+<style>
+body {
+	max-width: 50em;
+	margin: auto;
+	font-family: sans-serif;
+}
+</style>
+
+<h1>Agreement on HTML Ruby Markup</h1>
+
+<p>
+The W3C will specify extended HTML Ruby markup
```

Comment:
This should read:
```
The W3C has the option to specify extended HTML Ruby markup
```

since we can't commit to do this without proper AC Review of the charter and HTML Working Group approval.

Reply:
@plehegar Can we just switch “will” to “may” here rather than complicating the phrasing?

Reply:
sure

Reply:
See https://github.com/w3c/whatwg-coord/pull/15